GameServerLib/Logic/ObjectManager.cs

```diff
@@ -380,5 +380,27 @@ public bool TeamHasVisionOn(TeamId team, GameObject o)
 
             return false;
         }
+
+        public int CountUnitsAttackingUnit(AttackableUnit target)
+        {
+            int count = 0;
+            try
+            {
+                var enemyUnits = GetObjects();
+                var team = target.Team == TeamId.TEAM_BLUE ? TeamId.TEAM_PURPLE : TeamId.TEAM_BLUE;
```

Comment:
It would probably be better to use this extension method
https://github.com/LeagueSandbox/GameServer/blob/828e0f7714a644ed2aff87a0364a57da5800efe0/GameServerCore/Extensions.cs#L181